src/Evolution/Systems/Cce/Events/ObserveFields.hpp

```diff
@@ -130,7 +131,9 @@ std::string name() {
  * spin weighted quantities above (and time as the first column).
  *
  * All data will be written into the `observers::OptionTags::ReductionFileName`
- * file.
+ * file. If `WriteSynchronously` is true, then this will write the volume data
+ * immediately instead of sending it to the ObserverWriter to be written
+ * asynchronously.
  */
```

Comment:
Should we just check if running on 1 core then do sync output? Less thought required on the user part

Reply:
Yeah that's probably good. Even if it's slower, probably better than running out of memory